.go-version

```diff
@@ -1 +1 @@
-1.16.5
+1.16.8
```

Comment:
Let's use `1.17` then. WDYT ?

Reply:
Hi!, we'd have to go to `1.17.1` though since same vulnerability exists in `1.17`. But that works as well